spark/src/main/antlr/SqlBaseParser.g4

```diff
@@ -1096,7 +1096,7 @@ colPosition
     ;
 
 collateClause
-    : COLLATE collationName=stringLit
+    : COLLATE collationName=identifier
```

Comment:
why change it?

Reply:
I didn't change. It is coming from automated step in the build. let me verify again.

Reply:
Yes it is coming from the change in main spark.Shouldn't be an issue.